src/components/SideBar/ItemDrawer/ItemDrawer.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react'
+import { Header, Grid } from 'decentraland-ui'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import Drawer from 'components/Drawer'
+import AssetCard from 'components/AssetCard'
+import { Props, State } from './ItemDrawer.types'
+import './ItemDrawer.css'
+
+export default class ItemDrawer extends React.PureComponent<Props, State> {
+  state = {
+    listMode: false
+  }
+
+  renderGrid() {
+    const { items } = this.props
+    const { listMode } = this.state
+    let el = []
+
+    for (let i = 0; i < items.length; i += 3) {
+      let row = []
+
+      for (let j = i; j < i + 3; j++) {
+        const item = items[j]
+        if (!item) break
+
+        row.push(
+          <Grid.Column>
+            <AssetCard assetId={item.assetId} title={item.title} thumbnail={item.thumbnail} horizontal={listMode} />
+          </Grid.Column>
+        )
+      }
+
+      el.push(<Grid.Row>{row}</Grid.Row>)
+    }
+
+    return el
+  }
+
+  render() {
+    const { listMode } = this.state
+
+    return (
+      <div className="ItemDrawer">
+        <Header size="medium" className="title">
+          {t('itemDrawer.title')}
+        </Header>
+        <Drawer label="Category">
+          <Grid columns={1} padded="horizontally" className={`assetGrid ${listMode ? 'item-list' : 'item-grid'}`}>
```

Comment:
`assetGrid` is the only className (appart from Component class names) that uses camelCase. I think it should be `asset-grid`